apps/ejabberd/src/ejabberd_receiver.erl

```diff
@@ -142,27 +141,29 @@ init([Socket, SockMod, Shaper, MaxStanzaSize]) ->
 handle_call({starttls, TLSSocket}, _From, #state{parser = Parser} = State) ->
     NewParser = reset_parser(Parser),
     NewState = State#state{socket = TLSSocket,
-			   sock_mod = ejabberd_tls,
-			   parser = NewParser},
-    case ejabberd_tls:recv_data(TLSSocket, "") of
+         sock_mod = fast_tls,
+         parser = NewParser},
+    case fast_tls:recv_data(TLSSocket, "") of
         {ok, TLSData} ->
             {reply, ok, process_data(TLSData, NewState), ?HIBERNATE_TIMEOUT};
         {error, _Reason} ->
             {stop, normal, ok, NewState}
     end;
-handle_call({compress, ZlibSocket}, _From, #state{parser = Parser, c2s_pid = C2SPid} = State) ->
+handle_call({compress, ZlibSocket}, _From,
+  #state{parser = Parser, c2s_pid = C2SPid} = State) ->
     NewParser = reset_parser(Parser),
     NewState = State#state{socket = ZlibSocket,
-			   sock_mod = ejabberd_zlib,
-			   parser = NewParser},
+         sock_mod = ejabberd_zlib,
+         parser = NewParser},
     case ejabberd_zlib:recv_data(ZlibSocket, "") of
-	{ok, ZlibData} ->
-	    {reply, ok, process_data(ZlibData, NewState), ?HIBERNATE_TIMEOUT};
-	{error, inflate_size_exceeded} ->
-	    ?GEN_FSM:send_event(C2SPid, {xmlstreamerror, <<"XML stanza is too big">>}),
-	    {reply, ok, NewState, ?HIBERNATE_TIMEOUT};
-	{error, inflate_error} ->
-	    {stop, normal, ok, NewState}
+  {ok, ZlibData} ->
```

Comment:
Look like the whole block inside this case should be indented by additional 6 spaces. From some reason the indentation is only 2 spaces while it should be 8 inside this case.